sqlQueries/ConstructTables.sql

```diff
@@ -1,30 +1,36 @@
 CREATE TABLE Individuals (
-	IndividualID int,
+	IndividualID int NOT NULL AUTO_INCREMENT,
 	IndividualName varchar(255),
-	CurrentCompany varchar(255),
-	CurrentPosition varchar(255),
+    OriginalPostion varchar(255),
 	LinkedInUrl varchar(500),
-	Comments varchar(500)
+	Comments varchar(500),
+    PRIMARY KEY (IndividualID)
 );
 
 CREATE TABLE Companies (
-	CompanyID int,
-	CompanyName varchar(255)
+	CompanyID int NOT NULL AUTO_INCREMENT,
+	CompanyName varchar(255),
+    PRIMARY KEY (CompanyID)
 );
 CREATE TABLE EmployeeHistory (
-	IndividualID int,
-	CompanyID int,
+	IndividualID int NOT NULL,
+	CompanyID int NOT NULL,
 	PositionName varchar(255),
 	StartDate date,
-	EndDate date
+	EndDate date,
+    FOREIGN KEY (IndividualID) REFERENCES Individuals (IndividualID),
```

Comment:
Also, you will want indexes on every foreign key. Or you join queries will be super slow